models/asset.js

```diff
@@ -0,0 +1,110 @@
+const mongoose = require('../mongoose');
+const uuid = require('uuid');
+const Schema = mongoose.Schema;
+
+const AssetSchema = new Schema({
+
+  id: {
+    type: String,
+    default: uuid.v4,
+    unique: true,
+    index: true
+  },
+  url: {
+    type: String,
+    unique: true,
+    index: true
+  },
+  type: {
+    type: String,
+    default: 'article'
+  },
+  headline: String,
+  summary: String,
+  section: String,
+  subsection: String,
+  authors: [String],
+  publication_date: Date
+},{
+  _id: false,
+  versionKey: false,
+  timestamps: {
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  }
+});
+
+
+/**
+ * Search for assets. Currently only returns all.
+*/
+AssetSchema.statics.search = function(query) {
+
+  return Asset.find(query);
```

Comment:
My thought was that it's good practice to keep all database access in the model. This will lead to the fewest variations on database access and let us make intelligent decisions about what we want/don't want each item to do in one place.